### PR TITLE
Document addition of variable parameters to an include.

### PR DESCRIPTION
**Pull request by daviewales**
This fixes issue #3577.

### CODE REVIEW

**Comment by alfredxing** [comment]
:+1: Though we might want to use bold or italic instead of all caps?

File: site/_docs/templates.md [hunk 1]
```diff
@@ -286,10 +286,10 @@ root of your source directory. This will embed the contents of
   </p>
 </div>
 
-You can also pass parameters to an include:
+You can also pass parameters to an include. Note that the variable parameter is NOT surrounded with the usual liquid curly brackets:
 
 {% highlight ruby %}
-{% raw %}{% include footer.html param="value" %}{% endraw %}
+{% raw %}{% include footer.html param="value" variable-param=page.variable %}{% endraw %}
```
Comment (troyswanson) [inline site/_docs/templates.md]: It is totally possible that I'm wrong, but I *think* dashes are not allowed in Liquid variables...

Comment (daviewales) [inline site/_docs/templates.md]: Dashes seem to be working on my end.

**Comment by daviewales** [comment]
@alfredxing I've changed it from ALL CAPS to **strong**.

File: site/_docs/templates.md [hunk 2]
```diff
@@ -286,10 +286,10 @@ root of your source directory. This will embed the contents of
   </p>
 </div>
 
-You can also pass parameters to an include:
+You can also pass parameters to an include. Note that the variable parameter is **not** surrounded with the usual Liquid curly brackets:
```
Comment (parkr) [inline site/_docs/templates.md]: I don't understand "variable parameter" in this case. What do you think about:

> Omit the quotation marks if you would like to send a variable's value, and do not include the curly brackets.

Comment (daviewales) [inline site/_docs/templates.md]: That seems OK. I would probably say:

> Omit the quotation marks if you would like to send a variable's value. Liquid curly brackets should not be used here.

Comment (daviewales) [inline site/_docs/templates.md]: I've changed it to say

> Omit the quotation marks to send a variable's value. Liquid curly brackets should not be used here:

**Pull request merged**